src/core/server/http_resources/integration_tests/http_resources_service.test.ts

```diff
@@ -153,7 +152,6 @@ describe('http resources service', () => {
           res.renderHtml({
             body: '<html><p>Hi</p></html>',
             headers: {
-              'content-security-policy': "script-src 'unsafe-eval'",
               'content-type': 'text/html',
```

Comment:
Same as other comment above

```suggestion
              'content-security-policy': "script-src 'unsafe-eval'",
              'content-type': 'text/html',
```